internal/cluster/transition_test.go

```diff
@@ -3891,6 +3909,29 @@ var _ = Describe("Log Collection - refresh cluster", func() {
 		clusterId = strfmt.UUID(uuid.New().String())
 	})
 
+	AfterEach(func() {
```

Comment:
after each a new cluster is created? why?
I know that you just moved the code but i'm not sure why it's need to be done in after each

Reply:
That is how these tests are written: they run part of the setup and all the verifications inside the shared "AfterEach" block instead of inside the "It" block. I'd say it is a bit surprising, and it would probably be better to do that inside "It" and leave "AfterEach" only for cleanups. But I didn't want to change more things inside this patch.